src/coreclr/jit/liveness.cpp

```diff
@@ -1475,6 +1475,13 @@ void Compiler::fgComputeLifeLIR(VARSET_TP& life, BasicBlock* block, VARSET_VALAR
                     // or have a behavioral semantic that is undesirable to remove
                     break;
                 }
+#if defined(TARGET_ARM64)
```

Comment:
can you elaborate on why we are doing this? Why was the repro working in Debug and not in Release?